packages/medusa/src/migrations/1630868939283-product_variant_rank.ts

```diff
@@ -0,0 +1,14 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class productVariantRank1630868939283 implements MigrationInterface {
+    name = 'productVariantRank1630868939283'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "product_variant" ADD "variant_rank" integer NOT NULL`);
```

Comment:
the `NOT NULL` constraint here will break the migration for existing `product_variants` as these will have null values for the column by default - would simply leave out the NOT NULL constraint and add a default value of 0 instead